### PR TITLE
feat: add Contributor Distribution to repo page

### LINKED ISSUE #3711: Feature: add contribution distribution chart to Repo Pages
### Suggested solution

Our Contributor Distribution chart is a really good complement to the contributor chart in Repo Pages. This chart is a good visual way of showing all contributor activity based on PRs and identify the most active contributors in a repository.

image

For reference:
This chart is also used in Repository Insights > Activity
image

cc - @bdougie for visbility, @zeucapua is going to be working on this next 🫶

### PR DESCRIPTION
**Pull request by zeucapua**
## Description

Implement `<Activity />` component to show contributor distribution chart in repo pages under the "Contributor" tab. 

## Related Tickets & Documents
Closes #3711 

## Mobile & Desktop Screenshots/Recordings
image

## Steps to QA
1. Go to any repo page's "Contributor" tab
2. Check the Contribution Distribution chart.

## Tier (staff will fill in)

- [ ] Tier 1
- [x] Tier 2
- [ ] Tier 3
- [ ] Tier 4

## [optional] What gif best describes this PR or how it makes you feel?

lisa-simpson-gif-by-the-simpsons

### CODE REVIEW

**Comment by brandonroberts** [comment]
Nice! Could use a bit of spacing between the sections

image

**Review by bdougie — APPROVED** [review]
Also fine to approve as is to make it in, but notice that the chart doesn't behave well when resizing the screen

Rewatch GIF Recording - 2024-07-11 at 1 22 53 PM

**Comment by isabensusan** [comment]
Call that speed!! thank you 🙏 

+1 to @brandonroberts comment on the padding 

As I was reviewing responsiveness, I noticed the chart doesn't react well when resizing the screen, but overall behaves just fine when reloading on a specific breakpoint.

I'd say fine to merge as is, but let's fix that in a follow up PR

**Review by isabensusan — APPROVED** [review]

**Review by nickytonline — CHANGES_REQUESTED** [review]
One small change. The contributor distribution is stuck right to the top of the contributors table. It could use a little breathing room.

CleanShot 2024-07-11 at 16 35 45

Aside from that looks great on smaller screens.

CleanShot 2024-07-11 at 16 35 04

**Review by brandonroberts — APPROVED** [review]

File: pages/s/[org]/[repo]/contributors.tsx [hunk 1]
```diff
@@ -146,8 +146,10 @@ export default function RepoPageContributorsTab({ repoData, ogImageUrl }: RepoPa
             <TabList tabList={tabList} selectedTab={"contributors"} pageId={`/s/${repoData.full_name}`} />
           </div>
           <ClientOnly>
-            <Activity repositories={[repoData.id]} />
-            <Contributors repositories={[repoData.id]} defaultLayout="grid" />
+            <div className="flex flex-col gap-8">
```
Comment (nickytonline) [inline pages/s/[org]/[repo]/contributors.tsx:149]: nit: prefer grid for this use case.

```suggestion
<div className="grid gap-8">
```

**Review by nickytonline — APPROVED** [review]
🚢

**Review by nickytonline — APPROVED** [review]
🚢

**Pull request merged**